crates/nu-command/src/commands.rs

```diff
@@ -138,162 +138,160 @@ pub(crate) mod which_;
 pub(crate) mod with_env;
 pub(crate) mod wrap;
 
```

Comment:
Had to make some of these public, so that I can make a test context in nu_engine.